app/views/pages/edit.html.erb

```diff
@@ -0,0 +1,7 @@
+%= simple_form_for(boat) do |f| %>
+  <%= f.input :title %>
+  <%= f.input :description %>
+  <%= f.input :category %>
+  <%= f.input :price %>
```

Comment:
I would add the feature :
  <%= f.input :start_date %>
  <%= f.input :end_date %>